README.md

```diff
@@ -9,9 +9,14 @@ Spend less time learning about configuration and management of your app - and sp
 ## What's in this repo
 
 Code templates for easy Getting-Started experience with Google Cloud Code in Python, Java, Nodejs, Go and .NET Core.
-We support two IDEs: Visual Studio Code and IntelliJ.
+We support two IDEs: Visual Studio Code and IntelliJ (and other JetBrains IDEs).
```

Comment:
adding a small wording update in here too.

Reply:
Do you support Rider for .NET?

Reply:
No, not currently

Reply:
No, not currently